lib/param_parsing/terraform_version.go

```diff
@@ -0,0 +1,28 @@
+package param_parsing
+
+import (
+	"github.com/warrensbox/terraform-switcher/lib"
+	"os"
+	"strings"
+)
+
+const terraformVersionFileName = ".terraform-version"
+
+func GetParamsFromTerraformVersion(params Params) Params {
+	filePath := params.ChDirPath + "/" + terraformVersionFileName
+	if lib.CheckFileExist(filePath) {
+		logger.Infof("Reading configuration from %q", filePath)
+		content, err := os.ReadFile(filePath)
+		if err != nil {
+			logger.Errorf("Could not read file content at %q: %v", filePath, err)
```

Comment:
Maybe `logger.Fatalf()`? It looks like that other similar packages in this PR are fatalling on this kind of errors 🤔